free-3.md

```diff
@@ -1,43 +1,52 @@
-# Free project 3
+# Scheduling
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve users looking for a faster way to create and expand schedules. The DSL would "codify" scheduling by converting some text-based representation of the schedule into valid Google Calander or iCal format. This language might be a good way to meet the needs of people who enjoy the convenience of online schedules but find the process of filling it out and making changes or adding events very laborious. It might seem a bit silly at first to create an entire DSL for the sole purpose of saving a few minutes, but consider git. Github has both a terminal interface and a desktop alternative. Although some may consider the desktop interface to be limited in its functionality and less efficient than the terminal interface, others may find this more user-friendly and harder to mess up. Similarly, some may find terminal too hard to use or too easy to potentially mess up the repository. 
```

Comment:
Interesting, a project last semester in the class seems similar to what you suggest, you might want to check it out. https://github.com/aputman/calendarscript